tiatoolbox/models/dataset/info.py

```diff
@@ -61,16 +64,21 @@ class KatherPatchDataset(DatasetInfoABC):
     """Define a class for holding the Kather dataset information.
 
     Args:
-        save_dir_path (str or None): Path to directory containing the Kather
-          dataset. This is assumed to be the same form after the data is initially
-          downloaded. If the argument is `None`, the dataset will be downloaded
-          and extracted into the 'run_dir/download/Kather'.
-    Attributes
-        inputs (list): A list of paths where each path points to a sample image.
-        labels (list): A list of `int` where each is the label of the sample at
-          the same index.
-        label_names (dict): A dict indicates the possible associate name of each
-          label value.
+        save_dir_path (str or None):
+            Path to directory containing the Kather dataset. This is
+            assumed to be the same form after the data is initially
+            downloaded. If the argument is `None`, the dataset will be
+            downloaded and extracted into the 'run_dir/download/Kather'.
+
+    Attributes:
+        inputs (list):
+            A list of paths where each path points to a sample image.
+        labels (list):
+            A list of `int` where each is the label of the sample at the
```

Comment:
where each value corresponds to the label of the sample.

Reply:
Not quite sure if I follow. What is 'sample'? It is 'inputs'?

Reply:
@simongraham Are you able to clarify please? I am a little confused.